libs/mock-builder/src/lib.rs

```diff
@@ -337,3 +337,19 @@ macro_rules! execute_call {
 		$crate::execute::<CallIds<T>, _, _, _>(|| (), $input)
 	}};
 }
+
+/// Same as [`register_call`] but using an instanciable storage
+#[macro_export]
+macro_rules! register_call_i {
+	($f:expr) => {{
+		$crate::register::<CallIds<T, I>, _, _, _, _>(|| (), $f)
+	}};
+}
+
+/// Same as [`execute_call`] but using an instanciable storage
+#[macro_export]
+macro_rules! execute_call_i {
+	($input:expr) => {{
+		$crate::execute::<CallIds<T, I>, _, _, _>(|| (), $input)
+	}};
+}
```

Comment:
Shall be copied to: https://github.com/foss3/runtime-pallet-library/tree/main/mock-builder

Reply:
Done inf foss3 here: [86e6e69190ae8fdba28d714a6763aabbfc502d44](https://github.com/foss3/runtime-pallet-library/commit/86e6e69190ae8fdba28d714a6763aabbfc502d44)